runtime/src/lib.rs

```diff
@@ -394,8 +394,8 @@ parameter_types! {
     pub const TipFindersFee: Percent = Percent::from_percent(5);
     pub TipReportDepositBase: Balance = deposit(1, 0);
     pub BountyDepositBase: Balance = deposit(1, 0);
-    pub const BountyDepositPayoutDelay: BlockNumber = conf!(mainnet: 9 * DAYS, testnet: 6 * DAYS);
-    pub const BountyUpdatePeriod: BlockNumber = conf!(mainnet: 45 * DAYS, testnet: 35 * DAYS);
+    pub const BountyDepositPayoutDelay: BlockNumber = conf!(mainnet: 9 * DAYS, testnet: 6 * DAYS, devnet: DAYS);
```

Comment:
Minor: IMO `1 * DAYS` looks like more readable as well as other intervals such `1 * HOURS`.

Reply:
Agree. Updated the code to use `1 * DAYS` and similar intervals for better readability.